src/VisualStudio/CSharp/Impl/SemanticSearch/SemanticSearchToolWindowImpl.cs

```diff
@@ -100,7 +105,9 @@ public async Task<FrameworkElement> InitializeAsync(CancellationToken cancellati
 
         var vsUIShell = await vsUIShellProvider.GetValueAsync(cancellationToken).ConfigureAwait(false);
 
-        var textViewHost = CreateTextViewHost(vsUIShell);
+        var copilotUI = CreateCopilotUI();
+
+        var textViewHost = CreateTextViewHost(vsUIShell, copilotUI);
```

Comment:
changes in this file looks good to me, but we might want someone more familiar with UI stuff to take a look too

Reply:
Oh, the UI is a total throwaway hack. No need to review that :)